modules/mediawiki/files/cookbooks/rename-wiki.py

```diff
@@ -11,42 +22,54 @@
 }
 
 
-def get_db_cluster(oldwiki_db):
-    command = f"salt-ssh -E 'db131*' cmd.run 'mysql -e "USE mhglobal; SELECT wiki_dbcluster FROM cw_wikis WHERE wiki_dbname = \"{oldwiki_db}\"'"
-    result = subprocess.run(command, shell=True, stdout=subprocess.PIPE, text=True)
-    cluster_name = result.stdout.strip()
-    return db_clusters.get(cluster_name)
-
+def generate_salt_command(cluster: str, command: str) -> str:
+    return f'salt-ssh -E "{cluster}*" cmd.run "{command}"'
 
-def execute_salt_command(cluster, command):
-    salt_command = f"salt-ssh -E '{cluster}*' cmd.run '{command}'"
-    subprocess.run(salt_command, shell=True)
 
+def execute_salt_command(salt_command: str, shell: bool = False, stdout: Optional[int] = None, text: Optional[bool] = None) -> Optional[subprocess.CompletedProcess]:
+    response = input(f'EXECUTE (type c(continue), s(kip), a(bort): {salt_command}')
+    if response in ['c', 'continue']:
+        return subprocess.run(salt_command, shell=shell, stdout=stdout, text=text)
+    if response in ['s', 'skip']:
+        return None
+    sys.exit(1)  # noqa: R503
 
-def main():
-    parser = argparse.ArgumentParser(description='Executes the commands needed to rename wikis')
-    parser.add_argument('--oldwiki', required=True, help='Old wiki database name')
-    parser.add_argument('--newwiki', required=True, help='New wiki database name')
 
-    args = parser.parse_args()
+def get_db_cluster(oldwiki_db: str) -> str:
+    command = generate_salt_command('db131*', f'cmd.run "mysql -e \'SELECT wiki_dbcluster FROM mhglobal.cw_wikis WHERE wiki_dbname = "{oldwiki_db}" \' "')
```

Comment:
I think this should be:
command = generate_salt_command('db131*', f'cmd.run \'mysql -e "SELECT wiki_dbcluster FROM mhglobal.cw_wikis WHERE wiki_dbname = \'{oldwiki_db}\'" \'')